src/Pages/FarmTImeLine.js

```diff
@@ -60,17 +73,31 @@ function FarmTimeline() {
     setselectedEndDate(date);
   };
 
+  const [showTimeLine, setShowTimeLine] = useState(false);
+
   const handleCLick = () => {
-    console.log("locality,fieldId,cropSeason,activity" + locality, fieldId);
+    console.log(locality);
+
+    fetch(`http://localhost:8080/api/fieldCropCycles/${locality.value}`).then(
```

Comment:
can we have network calls in a different class? This can be added CropCycleClient.js

Reply:
ok RS.